front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -370,10 +370,17 @@ function Suggestions({
 
     const updateSuggestions = async () => {
       setSuggestionsStatus("loading");
+      // suggestions that are shown by default when no instructions are typed,
+      // are not considered as former suggestions. This way, the model will
+      // always generate tailored suggestions on the first input, which is preferable:
+      // - the user is more likely to be interested (since they likely saw the static suggestions before)
+      // - the model is not biased by static suggestions to generate new ones.
+      const formerSuggestions =
+        suggestions === STATIC_SUGGESTIONS ? [] : suggestions.slice(0, 2);
```

Comment:
Personally not a great fan of using reference comparison here since it's handled by a state down the line. Other alternative would be to augment the existing state to bundle a boolean, WDYT?